server/configuration_test.go

```diff
@@ -73,10 +73,10 @@ func TestOnConfigurationChange(t *testing.T) {
 		"same configuration": {
 			SetupAPI: func() *plugintest.API {
 				api := &plugintest.API{}
-				api.On("LoadPluginConfiguration", mock.Anything).Return(nil)
-				api.On("GetTeams").Return([]*model.Team{&model.Team{Id: teamId}}, nil)
+				api.On("LoadPluginConfiguration", mock.AnythingOfType("*main.configuration")).Return(nil)
```

Comment:
Using `mock.AnythingOfType("*main.configuration")` is a bit more specific then `mock.Anything`